airport/airport-pojo-bl/src/main/java/berlin/tablecodes/capex/Capex.java

```diff
@@ -0,0 +1,61 @@
+package berlin.tablecodes.capex;
+
+import ua.com.fielden.platform.entity.AbstractPersistentEntity;
+import ua.com.fielden.platform.entity.DynamicEntityKey;
+import ua.com.fielden.platform.entity.annotation.CompanionObject;
+import ua.com.fielden.platform.entity.annotation.CompositeKeyMember;
+import ua.com.fielden.platform.entity.annotation.DescRequired;
+import ua.com.fielden.platform.entity.annotation.DescTitle;
+import ua.com.fielden.platform.entity.annotation.DisplayDescription;
+import ua.com.fielden.platform.entity.annotation.IsProperty;
+import ua.com.fielden.platform.entity.annotation.KeyTitle;
+import ua.com.fielden.platform.entity.annotation.KeyType;
+import ua.com.fielden.platform.entity.annotation.MapEntityTo;
+import ua.com.fielden.platform.entity.annotation.MapTo;
+import ua.com.fielden.platform.entity.annotation.Observable;
+import ua.com.fielden.platform.entity.annotation.Title;
+import ua.com.fielden.platform.reflection.TitlesDescsGetter;
+import ua.com.fielden.platform.types.Money;
+import ua.com.fielden.platform.utils.Pair;
+
+/**
+ * Master entity object.
+ *
+ * @author Developers
+ *
+ */
+@KeyType(DynamicEntityKey.class)
+@KeyTitle("Capex")
+@CompanionObject(ICapex.class)
+@MapEntityTo
+@DescTitle("Description")
+@DisplayDescription
+@DescRequired
+public class Capex extends AbstractPersistentEntity<DynamicEntityKey> {
```

Comment:
Capex stands for "capital expenditure" (https://www.investopedia.com/terms/c/capitalexpenditure.asp). In many cases capital expenditures are captured as projects (e.g. a project for major upgrade of an airport security service). For this reason, it is more natural to model capex expenditures as entity `Project`, which later may get enhanced with additional information to better represent the financial aspects.

Reply:
I don't see an entity `Project` available.

Reply:
That because it was not created....